imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -12,6 +12,7 @@
 import org.motechproject.event.listener.EventRelay;
 import org.motechproject.event.listener.annotations.MotechListener;
 import org.motechproject.mds.query.SqlQueryExecution;
+import org.motechproject.metrics.service.Timer;
```

Comment:
I think we might have two Timer like classes now.  org.apache.commons.lang.time.StopWatch was introduced during the MCTS web service work.  Ideally we would consolidate, I'm not sure if the time it takes to do that is worth it now.